src/MyEventEmitter.js

```diff
@@ -1,12 +1,84 @@
 'use strict';
 
-export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+const http = require('http');
+const server = new http.Server();
+
+class MyEventEmitter {
+  constructor() {
+    this.events = {};
+  }
+
+  on(eventName, listener) {
+    if (eventName in this.events) {
+      this.events[eventName].push(listener);
+    } else {
+      this.events[eventName] = [listener];
+    }
```

Comment:
make sense to use ternary operator here as it will make a code shorter 
check other places